namiblii/templates/liiweb/home.html

```diff
@@ -0,0 +1,103 @@
+{% extends "liiweb/home.html" %}
+{% load i18n static %}
+{% block hero-logos %}
+  <div class="d-flex justify-content-between">
+    <img height="26"
+         class="me-4"
+         src="{% static 'images/africanlii-logo.svg' %}"
+         alt="africanlii-logo"/>
+    <img height="26"
+         src="{% static 'images/laws-africa-logo.png' %}"
+         alt="laws-logo"/>
+    <img height="26"
+         src="{% static 'images/lac_logo_full.jpg' %}"
+         alt="lac-logo"/>
+  </div>
+{% endblock %}
+{% block second-column-content %}
+  <div>
+    <strong>{% trans 'About' %}</strong>
+  </div>
+  <p>
+    {% blocktrans trimmed %}
+      NamibLII is a project of the Law Reform and Development Commission and a member of the global Free Access to Law Movement and the AfricanLII community.
+  {% endblocktrans %}
+  </p>
+  <div class="d-flex justify-content-between mb-4 mb-lg-4">
+    <img height="26"
+         class="me-4"
+         src="{% static 'images/africanlii-logo.svg' %}"
+         alt="africanlii-logo"/>
+    <img height="26"
+         src="{% static 'images/laws-africa-logo.png' %}"
+         alt="laws-logo"/>
+    <img height="26"
+         src="{% static 'images/lac_logo_full.jpg' %}"
+         alt="laws-logo"/>
+  </div>
+  <ul class="nav nav-tabs about-tabs d-flex justify-content-center justify-content-lg-start"
```

Comment:
I've just merged a change by Sandrava that puts the Twitter tab first. Rather than having to duplicate that everywhere, can we make this a part of the base template and make the About text above a separate block? That way we can override just the about text and the social media stuff comes from the parent template.